syndeo-template/src/main/java/com/google/cloud/syndeo/dlq/pubsub/PubsubDlqSchemaTransformProvider.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.syndeo.dlq.pubsub;
+
+import com.google.auto.service.AutoService;
+import com.google.auto.value.AutoValue;
+import com.google.cloud.syndeo.dlq.pubsub.PubsubDlqSchemaTransformProvider.PubsubDlqWriteConfiguration;
+import com.google.cloud.syndeo.transforms.TypedSchemaTransformProvider;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubClient;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubIO;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage;
+import org.apache.beam.sdk.schemas.transforms.SchemaTransform;
+import org.apache.beam.sdk.schemas.transforms.SchemaTransformProvider;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.PTransform;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.PCollectionRowTuple;
+import org.apache.beam.sdk.values.Row;
+import org.checkerframework.checker.initialization.qual.Initialized;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.UnknownKeyFor;
+
+@AutoService(SchemaTransformProvider.class)
+public class PubsubDlqSchemaTransformProvider
+    extends TypedSchemaTransformProvider<PubsubDlqWriteConfiguration> {
+
+  @Override
+  public Class<PubsubDlqWriteConfiguration> configurationClass() {
+    return PubsubDlqWriteConfiguration.class;
+  }
+
+  @Override
+  public SchemaTransform from(PubsubDlqWriteConfiguration configuration) {
+    return new PubsubDlqSchemaTransform(configuration);
+  }
+
+  @Override
+  public @UnknownKeyFor @NonNull @Initialized String identifier() {
+    return "syndeo:schematransform:com.google.cloud:pubsub_dlq_write:v1";
+  }
+
+  @Override
+  public @UnknownKeyFor @NonNull @Initialized List<@UnknownKeyFor @NonNull @Initialized String>
+      inputCollectionNames() {
+    return Collections.singletonList("input");
```

Comment:
```suggestion
    return Collections.singletonList("errors");
```

Since the input collectin name is "errors", we should change that here as well.

Reply:
done. not sure how i missed this.